rust/scx_rustland_core/Cargo.toml

```diff
@@ -15,12 +15,17 @@ include = [
 
 [dependencies]
 anyhow = "1.0"
-libbpf-rs = "0.22.0"
+libbpf-rs = "0.22"
```

Comment:
Now that libbpf-rs 0.23 is out should we depend on this version?

Reply:
@htejun not sure if you were extremely fast, or you were doing exactly this change while I was doing my review... in any case using libbpf-rs 0.23 looks good to me :)

Reply:
Yeah, I was already updating. :)